Lib/fontTools/varLib/featureVars.py

```diff
@@ -614,6 +626,29 @@ def buildConditionTable(axisIndex, filterRangeMinValue, filterRangeMaxValue):
     return ct
 
 
+def findFeatureVariationRecord(featureVariations, conditionTable):
+    """Find a FeatureVariationRecord that has the same conditionTable."""
+    if featureVariations.Version != 0x00010000:
+        raise VarLibError(
+            "Unsupported FeatureVariations table version: "
+            f"0x{featureVariations.Version:08x} (expected 0x00010000)."
+        )
+
+    for fvr in featureVariations.FeatureVariationRecord:
+        if fvr.ConditionSet.ConditionCount != len(conditionTable):
+            continue
+        for ct1, ct2 in zip(conditionTable, fvr.ConditionSet.ConditionTable):
+            if (
+                ct1.Format == ct2.Format
+                and ct1.AxisIndex == ct2.AxisIndex
+                and ct1.FilterRangeMinValue == ct2.FilterRangeMinValue
+                and ct1.FilterRangeMaxValue == ct2.FilterRangeMaxValue
+            ):
```

Comment:
I believe (though haven't tested) that you could simply compare ct1 == ct2 because BaseTable implements `__eq__` that compares attributes by value, maybe try see if that works

Reply:
yes, I think it should work, see https://github.com/fonttools/fonttools/blob/e3cde46e69074f75462f3ca1bfeec736f2b559ee/Lib/fontTools/ttLib/tables/otBase.py#L1166-L1173